README.md

```diff
@@ -54,6 +72,7 @@ This config requires `vaadin/prettier` that must be added the very last i.e. aft
   },
   "extends": [
     "vaadin/typescript",
+    "vaadin/typescript-requireing-type-checking",
```

Comment:
```suggestion
    "vaadin/typescript-requiring-type-checking",
```

Also, what about reducing the length a bit, like: `vaadin/typescript-requiring-typecheck`?

Reply:
We could do that 👍 . I just wanted it to be aligned with `typescript-eslint/recommended-requiring-type-checking`.

Reply:
Agreed internally not to change it. Let's keep it consistent with `typescript-eslint`.